client/src/config.js

```diff
@@ -1,9 +1,9 @@
 const config = {
   // OAuth info (copied from the FusionAuth admin panel)
-  clientID: '48f671cc-1627-41b5-aed1-260b80a9f742',
-  clientSecret: 'pqQj3uOj7q7BLe0ImvgSxCvKUTuczAth1vjAmkntve8',
+  clientId: '85a03867-dccf-4882-adde-1a79aeec50df',
+  clientSecret: '6HvlAe28V4PL9OEKJ0VlZ6ZsEAluEsZHVFYKuoLAr1U',
```

Comment:
Should we generate the client secret so it is different for each user of this code, and possibly the clientId? Or at least mention that the user of this should change this value if they are using it for anything but the example.